cmd/minikube/cmd/start_flags.go

```diff
@@ -212,6 +217,12 @@ func initNetworkingFlags() {
 	startCmd.Flags().String(serviceCIDR, constants.DefaultServiceCIDR, "The CIDR to be used for service cluster IPs.")
 	startCmd.Flags().StringArrayVar(&config.DockerEnv, "docker-env", nil, "Environment variables to pass to the Docker daemon. (format: key=value)")
 	startCmd.Flags().StringArrayVar(&config.DockerOpt, "docker-opt", nil, "Specify arbitrary flags to pass to the Docker daemon. (format: key=value)")
+
+	// generic
+	startCmd.Flags().String(genericIPAddress, "", "IP address (generic)")
```

Comment:
We should validate that all of these flags are set and exit with a helpful error message if the user has forgotten anything

Reply:
I did that, but it was lost during the rebase. It should at least give an error if forgetting to provide an address ?

The other parameters are optional, by default it will connect as root on port 22 and assume keys are in ssh-agent